dashboard/app/controllers/api/v1/pd/foorm/library_questions_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Api::V1::Pd::Foorm::LibraryQuestionsController < ApplicationController
+  # POST api/v1/pd/foorm/library_questions/validate_library_question
+  def validate_library_question
+    authorize! :validate_library_question, :pd_foorm
+
+    library_question = Foorm::LibraryQuestion.new(question: params[:question].to_json)
+    library_question.validate_library_question
+
+    if library_question.errors.empty?
+      return render status: 200, json: {}
+    else
+      return render status: 500, json: {error: library_question.errors}
```

Comment:
are we surfacing these to the user?

Reply:
Yeah, if you click the "Validate" button in the UI, you are told what errors you have (or not).